src/main/java/com/spotify/github/v3/clients/GitHubClient.java

```diff
@@ -362,6 +366,15 @@ public ChecksClient createChecksClient(final String owner, final String repo) {
     return ChecksClient.create(this, owner, repo);
   }
 
+  /**
+   * Create organisation API client
+   *
+   * @return organisation API client
+   */
+  public OrganisationClient createOrganisationClient() {
```

Comment:
Would it make sense to pass the org name into `OrganizationClient` (like we do in a similar way e.g. in `createChecksClient`), so that we don't have to pass the org name into each call in the new class?

Reply:
I think the only reason I left it was in case you were using a single class to manipulate data across multiple orgs. I'm not entirely sure what the best way to handle this one is, I'm happy to update it either way :)

Reply:
All the other client classes take their "global" arguments in the constructor, so I guess it's most consistent to also do that here.

Reply:
Okay sweet, I'll update that now :)